megamek/src/megamek/common/options/Quirks.java

```diff
@@ -222,36 +222,47 @@ public static boolean isQuirkLegalFor(IOption quirk, Entity en) {
-                            QUIRK_NEG_EM_INTERFERENCE_WHOLE,                        QUIRK_NEG_POOR_PERFORMANCE,
-                            QUIRK_NEG_HARD_PILOT,       QUIRK_NEG_POOR_TARG_S,      QUIRK_NEG_POOR_TARG_M,
-                            QUIRK_NEG_POOR_TARG_L,      QUIRK_NEG_POOR_WORK,        QUIRK_NEG_PROTOTYPE,
-                            QUIRK_NEG_SENSOR_GHOSTS,    QUIRK_POS_UBIQUITOUS_IS,    QUIRK_POS_UBIQUITOUS_CLAN);
+                            QUIRK_POS_ANTI_AIR, 		QUIRK_POS_BATTLE_COMP, 		QUIRK_POS_EASY_MAINTAIN,
+                            QUIRK_POS_FAST_RELOAD, 		QUIRK_POS_GOOD_REP_1, 		QUIRK_POS_GOOD_REP_2,
+                            QUIRK_POS_IMP_COM, 			QUIRK_POS_IMPROVED_SENSORS,
+                            QUIRK_POS_IMP_TARG_S, 		QUIRK_POS_SEARCHLIGHT,
+                            QUIRK_POS_IMP_TARG_M, 		QUIRK_POS_IMP_TARG_L, 		QUIRK_POS_LOW_PROFILE,
+                            QUIRK_NEG_BAD_REP_IS, 		QUIRK_NEG_BAD_REP_CLAN, 	QUIRK_NEG_DIFFICULT_MAINTAIN,
+                            QUIRK_NEG_NON_STANDARD,		QUIRK_NEG_POOR_PERFORMANCE, QUIRK_NEG_HARD_PILOT, 		
+                            QUIRK_NEG_POOR_TARG_S,      QUIRK_NEG_POOR_TARG_M, 		QUIRK_NEG_POOR_TARG_L, 		
+                            QUIRK_NEG_POOR_WORK,        QUIRK_NEG_PROTOTYPE,        QUIRK_NEG_SENSOR_GHOSTS, 	
+                            QUIRK_POS_UBIQUITOUS_IS,    QUIRK_POS_UBIQUITOUS_CLAN);
             }
         }
 
         if (en instanceof BattleArmor) {
             return quirk.isAnyOf(
-                    QUIRK_NEG_HARD_PILOT,           QUIRK_NEG_BAD_REP_IS,       QUIRK_NEG_BAD_REP_CLAN,
-                    QUIRK_NEG_DIFFICULT_MAINTAIN,   QUIRK_NEG_PROTOTYPE,        QUIRK_NEG_POOR_TARG_S,
-                    QUIRK_NEG_POOR_TARG_M,          QUIRK_NEG_POOR_TARG_L,      QUIRK_NEG_POOR_WORK,
-                    QUIRK_NEG_SENSOR_GHOSTS,        QUIRK_POS_EASY_MAINTAIN,    QUIRK_POS_IMP_COM,
-                    QUIRK_POS_EASY_PILOT,           QUIRK_POS_FAST_RELOAD,      QUIRK_NEG_EM_INTERFERENCE_WHOLE,
-                    QUIRK_NEG_NON_STANDARD);
+                    		QUIRK_POS_EASY_MAINTAIN,	QUIRK_POS_EASY_PILOT,		QUIRK_POS_GOOD_REP_1,
+                    		QUIRK_POS_GOOD_REP_2,       QUIRK_POS_IMP_COM,      	QUIRK_POS_RUGGED_1,
+		                    QUIRK_POS_RUGGED_2,         QUIRK_POS_UBIQUITOUS_IS,	QUIRK_POS_UBIQUITOUS_CLAN,
+		                    QUIRK_NEG_BAD_REP_IS,       QUIRK_NEG_BAD_REP_CLAN,     QUIRK_NEG_DIFFICULT_MAINTAIN,
+		                    QUIRK_NEG_HARD_PILOT,       QUIRK_NEG_ILLEGAL_DESIGN,   QUIRK_NEG_NON_STANDARD,
+		                    QUIRK_NEG_OBSOLETE,         QUIRK_NEG_POOR_SEALING,     QUIRK_NEG_POOR_TARG_S,
+		                    QUIRK_NEG_POOR_TARG_M,      QUIRK_NEG_POOR_TARG_L,      QUIRK_NEG_POOR_WORK,
+		                    QUIRK_NEG_PROTOTYPE,        QUIRK_NEG_SENSOR_GHOSTS);
         }
 
-        if (en instanceof Aero) {
+        if (en instanceof Aero || en instanceof FixedWingSupport) {
```

Comment:
FixedWingSupport is a subclass of Aero. So `en instanceof Aero` already includes them.